javagen/src/main/resources/MultipartFormDataHelper.java

```diff
@@ -156,8 +176,30 @@ public MultipartFormDataHelper end() {
         return this;
     }
 
+    private void writeFileField(String fieldName, BinaryData file, String filename) {
+        // Multipart preamble
+        String fileFieldPreamble = partSeparator
+                + CRLF + "Content-Disposition: form-data; name=\"" + fieldName
+                + "\"; filename=\"" + filename + "\""
+                + CRLF + "Content-Type: application/octet-stream" + CRLF + CRLF;
+        byte[] data = fileFieldPreamble.getBytes(encoderCharset);
+        appendBytes(data);
+
+        // Writing the file into the request as a byte stream
+        requestLength += file.getLength();
+        requestDataStream = new SequenceInputStream(requestDataStream, file.toStream());
+
+        // CRLF
+        data = CRLF.getBytes(encoderCharset);
+        appendBytes(data);
+    }
+
     private void appendBytes(byte[] bytes) {
         requestLength += bytes.length;
         requestDataStream = new SequenceInputStream(requestDataStream, new ByteArrayInputStream(bytes));
     }
+
+    private String normalizeAscii(String text) {
+        return Normalizer.normalize(text, Normalizer.Form.NFD).replaceAll("[^\\x00-\\x7F]", "");
+    }
```

Comment:
This is for better safety of the request. Generally, RFC ask the MIME be US-ASCII.